airbyte-integrations/connectors/destination-postgres-strict-encrypt/src/test/resources/expected_spec.json

```diff
@@ -143,7 +143,7 @@
               },
               "client_key_password": {
                 "type": "string",
-                "title": "Client key password (Optional)",
```

Comment:
These weren't updated when we removed the optionals so the spec test was failing.